### ISSUE TITLE
Cirrus Background doesn't work

### ISSUE THREAD

**Issue by learnspyral** [voyanttools/Voyant#77]
The code for setting the background colour doesn't work.
myCorpus.tool("Cirrus", {style: "background: blue"});
Is this no longer possible?

**Comment by ajmacdonald** [comment]
Good catch. This functionality was lost at a certain point and needs to be re-included.

**Comment by ajmacdonald** [comment]
Note to self: https://github.com/voyanttools/Voyant/blob/master/src/main/webapp/app/panel/Cirrus.js#L429
